semver/is_semver_comparator.ts

```diff
@@ -13,18 +11,8 @@ import { ALL, NONE } from "./constants.ts";
  * Adds a type assertion if true.
  * @param value The value to check if its a SemVerComparator
  * @returns True if the object is a SemVerComparator otherwise false
+ * @deprecated (will be removed in 0.212.0) Use {@linkcode isComparator} instead.
  */
 export function isSemVerComparator(value: unknown): value is SemVerComparator {
-  if (value === null || value === undefined) return false;
-  if (value === NONE) return true;
-  if (value === ALL) return true;
-  if (Array.isArray(value)) return false;
-  if (typeof value !== "object") return false;
-  const { operator, semver, min, max } = value as SemVerComparator;
-  return (
-    isValidOperator(operator) &&
-    isSemVer(semver) &&
-    isSemVer(min) &&
-    isSemVer(max)
-  );
+  return isComparator(value);
 }
```

Comment:
```suggestion
export const isSemVerComparator = isComparator;
```
Sorry, should've spotted this earlier.

Reply:
done